### ISSUE TITLE
Missing support for version 3

### ISSUE THREAD

**Issue by ngoldack** [paulvarache/vscode-taskfile#6]
Hi I really like your extension, but the new [version "3"](https://github.com/go-task/task/releases/tag/v3.0.0) is missing.

![image](https://user-images.githubusercontent.com/1354182/90906653-48120800-e3d2-11ea-84df-11d96756dfe0.png)

**Comment by KROSF** [comment]
[Here](https://gist.github.com/KROSF/c5435acf590acd632f71bb720f685895) the schema for version 3

**Comment by xmlking** [comment]
looking forward new build/release. last release was 2/12/2020, 1:32:20 AM
as of now, I am getting 
```
Error running command taskfile.run: Task is not installed. This is likely caused by the extension that contributes taskfile.run.
```
I already install `task` with brew`brew install go-task/tap/go-task` 
I also tried `Install Task Binary` but no luck 
![image](https://user-images.githubusercontent.com/627951/105094386-11abe800-5a59-11eb-8df4-81ee4f5d033b.png)

**Comment by mhmdio** [comment]
Confirm @xmlking issue, same here

**Comment by paulvarache** [comment]
Just released 0.0.5 with support with Taksfile v3
